pkg/volume/azure_file/azure_file.go

```diff
@@ -222,11 +226,12 @@ func (b *azureFileMounter) SetUpAt(dir string, fsGroup *int64) error {
 	} else {
 		os.MkdirAll(dir, 0700)
```

Comment:
You should use the `dir_mode` value if defined, otherwise defaultFileMode when the root folder is created.

Reply:
actually here `dir` is the k8s path in the agent node, e.g. `/var/lib/kubelet/plugins/kubernetes.io/azure-disk/mounts/b1246717734`, not the container mount path, k8s would then use volume mapping between `dir` and container mount path. So `dir_mode` is different here, we should not set as `dir_mode` in `mountOptions`.